src/architect/project/project.utils.ts

```diff
@@ -0,0 +1,229 @@
+import fs from 'fs-extra';
+import inquirer from 'inquirer';
+import yaml from 'js-yaml';
+import fetch from 'node-fetch';
+import path from 'path';
+import untildify from 'untildify';
+import { buildSpecFromPath, ComponentSpec } from '../../';
+import AppService from '../../app-config/service';
+import { EnvironmentSpecValue } from '../../dependency-manager/spec/resource-spec';
+import { Dictionary } from '../../dependency-manager/utils/dictionary';
+
+const download = require('download-git-repo');
```

Comment:
what is the benefit of this package as opposed to shelling out via execa?

`execa(git clone ...)`

Reply:
@mtgoncalves1 let's switch to using `execa`, unless there was additional benifits from `download-git-repo` package. 

UX portions later in the script will also need to leverage `execa` for linking and other things

Reply:
Sorry, that was already addressed.